src/sage/schemes/hyperelliptic_curves/hypellfrob.pyx

```diff
@@ -68,21 +73,17 @@ def interval_products(M0, M1, target):
     r"""
     Given a matrix `M` with coefficients linear polynomials over `\ZZ/N\ZZ` and
     a list of integers `a_0 < b_0 \le a_1 < b_1 \le \cdots \le a_n < b_n`
-    compute the matrices
-    ``\prod_{t = a_i + 1}^{b_i} M(t)``
-    for `i = 0` to `n`.
-
-    This is a wrapper for code in the ``hypellfrob`` package.
+    compute the matrices `\prod_{t = a_i + 1}^{b_i} M(t)` for `i = 0` to `n`.
 
     INPUT:
 
-    - ``M0``, ``M1`` -- matrices over `\ZZ/N\ZZ`, so that `M = M0 + M1*x`
+    - ``M0``, ``M1`` -- matrices over `\ZZ/N\ZZ`, so that `M = M0 + M1\times x`
     - ``target`` -- a list of integers
```

Comment:
```suggestion
    - ``target`` -- a list of integers `[a_0, b_0, \cdots, a_n, b_n]`
```
(will anyone read this?)

Reply:
Not an interesting read, but still visible.

Fixed with
```
a list of integers `a_0, b_0, \dots, a_n, b_n`
```